ansible/setup-nodes.yml

```diff
@@ -4,4 +4,9 @@
   become: yes
 
   roles:
-     - role: lablabs.rke2
\ No newline at end of file
+     - role: upgrade_machine
+     - role: lablabs.rke2
+       vars: 
+         rke2_download_kubeconf: true
+         rke2_download_kubeconf_file_name: rke2.yaml
+         rke2_download_kubeconf_path: /tmp
```

Comment:
We should update the readme to get users to copy this into `/etc` as per the current instructions